### DISCUSSION TITLE
Samba SMB v1 Activation

### DISCUSSION THREAD

**Discussion by mercenaruss** [nextcloud/all-in-one#2121] [Questions] [answered]
Hi,
I have some network share on another server what a user require to connect like External Storage,but unfortunately is SMB v1 and i'm unable to connect to NextCloud AIO,is any way to enable SMBv1?

**Comment by szaimen** [comment, marked as answer]
Hi, we will not add this since SMB v1 is deprecated and insecure since many years.

You should rather look into how to enable SMB v2 or better v3 on your SMB-server.
